.csslintrc

```diff
@@ -1 +1 @@
---ignore=box-sizing,compatible-vendor-prefixes,known-properties,important,unqualified-attributes
+--ignore=box-sizing,compatible-vendor-prefixes,known-properties,important,unqualified-attributes,duplicate-background-images
```

Comment:
Interesting.  I've been finding the CSS linting much less useful than the JS linting.  Still catches the odd thing though.